main/install/db_main.sql

```diff
@@ -226,6 +226,19 @@ CREATE TABLE IF NOT EXISTS course_field (
 );
 
 --
+-- Table structure for table course_field_optionss
```

Comment:
New tables should go at the *end* of the db_main.sql file, so we can keep track of the order in which we add them to the file. I mentioned this in our small meeting.